proxy/hdrs/HdrHeap.cc

```diff
@@ -959,6 +960,9 @@ HdrHeap::unmarshal(int buf_length, int obj_type, HdrHeapObjImpl **found_obj, Ref
     obj_data = obj_data + obj->m_length;
   }
 
+  if (obj_found == false)
```

Comment:
I'd prefer `if (!obj_found)` - IMHO it's never a good idea to compare to a boolean literal.